maths/maclaurin_sin.py

```diff
@@ -0,0 +1,56 @@
+"""
+https://en.wikipedia.org/wiki/Taylor_series#Trigonometric_functions
+"""
+from math import factorial
+
+
+def maclaurin_sin(theta: float, accuracy: int = 30) -> float:
+    """
+    Finds the maclaurin approximation of sin
+
+    :param theta: the angle to which sin is found
+    :param accuracy: the degree of accuracy wanted minimum ~ 1.5 theta
+    :return: the value of sine in radians
+
+    >>> maclaurin_sin(10)

```

Comment:
Please add this test (or similar):
```suggestion
    >>> from math import isclose, sin
    >>> all(isclose(maclaurin_sin(x), sin(x)) for x in range(-25, 25))
    True
    >>> maclaurin_sin(10)
```
https://docs.python.org/3/library/math.html#math.isclose